src/components/Map.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import '../index.css';
+import * as L from 'leaflet'
```

Comment:
This is fine, especially for such a simple project, but in terms of what we normally use, we've lately started to turn to Vector Tiles / Mapbox GL JS as our go-to mapping solution. This is primarily due to the combination's higher performance for large vector datasets.